src/storage.ts

```diff
@@ -393,6 +405,12 @@ export class Storage extends Service {
    * @param {StorageOptions} [options] Configuration options.
    */
   constructor(options: StorageOptions = {}) {
+    const AUTO_RETRY_DEFAULT = true;
+    const MAX_RETRY_DEFAULT = 3;
```

Comment:
Was this constant being used already? 3 seems way too low.

Reply:
Yeah this is the current default